ivy/functional/ivy/statistical.py

```diff
@@ -1260,3 +1260,17 @@ def einsum(
 
     """
     return current_backend(operands[0]).einsum(equation, *operands, out=out)
+
```

Comment:
This is not part of the task, and of currently we are not adding any more functions to the Ivy API